DESCRIPTION

```diff
@@ -64,7 +64,9 @@ Imports:
     plotly,
     urltools,
     geojsonsf,
-    tibble
+    tibble,
+    nhdplusTools,
+    arcgis
```

Comment:
There is an issue with loading the arcgis package since it is not available on CRAN anymore. We can still use it, but it needs to go under Remotes

`r
Remotes: 
    github::R-ArcGIS/arcgis
`

Reply:
@kathryn-willi I haven't been able to resolve the check() dependency issues with arcgislayers yet. Do you know if there a workaround/alternative to using that package?